<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>avion</class>
 <widget class="QDialog" name="avion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="AVION">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>531</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacité</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_id">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_capacite">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_Ajouter">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:lime;</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>247</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/logo 1.3.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>481</x>
       <y>213</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/logo4.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/image1.jpg</pixmap>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_etat">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Fonctionnel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Endommagé</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En réparration</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cb_type">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>72</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>jet privé</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>avion de tourisme</string>
      </property>
     </item>
    </widget>
    <zorder>label_16</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>le_id</zorder>
    <zorder>le_capacite</zorder>
    <zorder>pb_Ajouter</zorder>
    <zorder>label_14</zorder>
    <zorder>label_15</zorder>
    <zorder>cb_etat</zorder>
    <zorder>cb_type</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacité</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id_m">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_capacite_m">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_type_m">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_etat_m">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:blue;
</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/image2.jpg</pixmap>
     </property>
    </widget>
    <zorder>label_17</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
    <zorder>label_11</zorder>
    <zorder>lineEdit_id_m</zorder>
    <zorder>lineEdit_capacite_m</zorder>
    <zorder>lineEdit_type_m</zorder>
    <zorder>lineEdit_etat_m</zorder>
    <zorder>pushButton_modifier_2</zorder>
   </widget>
   <widget class="QWidget" name="Afficher">
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_Afficher_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_imprimer">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_trier">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>180</y>
       <width>72</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Capacité</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Etat</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>180</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>230</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rechercher</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>230</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_docExel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>130</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Document Excel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/logo4.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>581</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/image6.jpg</pixmap>
     </property>
    </widget>
    <widget class="QTableView" name="tab_afficher">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <zorder>label_13</zorder>
    <zorder>pushButton_Afficher_2</zorder>
    <zorder>pushButton_imprimer</zorder>
    <zorder>comboBox_trier</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>pushButton_docExel</zorder>
    <zorder>label_18</zorder>
    <zorder>tab_afficher</zorder>
   </widget>
   <widget class="QWidget" name="Supprimer">
    <attribute name="title">
     <string>Supprimer</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>261</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/logo4.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-8</y>
       <width>531</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../pp/Nouveau dossier/3231ca3a13365475.jpg</pixmap>
     </property>
    </widget>
    <zorder>label_19</zorder>
    <zorder>label_12</zorder>
    <zorder>label_5</zorder>
    <zorder>pushButton</zorder>
    <zorder>lineEdit</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
